code_of_conduct.html

```diff
@@ -0,0 +1,128 @@
+  <link rel="stylesheet" type="text/css" href="http://cfgmgmtcamp.eu/stylesheets/pygment_trac.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="http://cfgmgmtcamp.eu/stylesheets/print.css" media="print" />
+  <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.css" />
+  <link rel="shortcut icon" href="http://cfgmgmtcamp.eu/favicon.ico">
+  <!--[if lte IE 8]>
+  <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.ie.css" />
+  <![endif]-->
+  <!--[if lt IE 9]>
+  <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+  <script src="http://cdn.leafletjs.com/leaflet-0.6.4/leaflet.js"></script>
+  <title>Code of Conduct - Config Management Camp 2015, Gent</title>
+</head>
+
+<body>
+  <nav>
+    <a href="#" id="menu-icon">&#x2261;</a>
+    <ul>
+      <li><a href="http://cfgmgmtcamp.eu/#about">About</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/#registration">Registration</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/#sponsors">Sponsors</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/#route">Location</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/#contributions">Contributions</a></li>
+      <li><a href="code_of_conduct.html">Code of Conduct</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/2014/">2014</a></li>
+    </ul>
+  </nav>
+
+  <div id="container">
+    <header>
+      <div class="inner">
+        <h1>#cfgmgmtcamp</h1>
+        <div class="camp-info">
+          <h2><strong>2</strong> and <strong>3</strong> February 2015</h2>
+          <h2>Gent, Belgium </h2>
+        </div>
+      </div>
+    </header>
+
+
+    <div class="inner">
+      <section id="content">
+        <h1>Code of Conduct</h1>
+
+        <p>
+          Configuration Management Camp is a community conference intended for networking and collaboration in the community of people interested in configuration management.
+        </p>
+
+        <p>
+          We value the participation of each member of the community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored or not.
+        </p>
+
+        <p>
+          Exhibitors, speakers, sponsors, staff and all other attendees at Configuration Management Camp are subject to this Code of Conduct. We are dedicated to providing a harassment-free experience for everyone, and we do not tolerate harassment of participants in any form.
+        </p>
+
+        <p>
+          We ask you to be considerate of others and behave professionally and respectfully to all other participants. Remember that sexual language and imagery is not appropriate for any event venue, including talks. Participants violating these rules may be sanctioned or expelled from the event without a refund at the discretion of the conference organizers.
```

Comment:
A refund doesn't really make sense for CMCamp since it's a free event. However, we could say "without compensation" (for travel, hotel, etc) instead?

Doesn't matter much either way - don't block merging the PR for this.